### PR TITLE
Truncate title for work process

### PR DESCRIPTION
**Pull request by FerPerales**
Asana ticket: https://app.asana.com/0/1203289004376659/1204828166480417/f

image

### CODE REVIEW

File: app/dashboards/work_process_dashboard.rb [hunk 1]
```diff
@@ -50,6 +52,6 @@ class WorkProcessDashboard < Administrate::BaseDashboard
   COLLECTION_FILTERS = {}.freeze
 
   def display_resource(work_process)
-    work_process.title
+    work_process.title.truncate(MAX_TITLE_LENGTH)
```
Comment (sarahraqueld) [inline app/dashboards/work_process_dashboard.rb:55]: I see the `title` field is not NOT NULL on the db, which means some work processes will have a nil title. We'll have an undefined method error then, should we protect ourselves against it?

Comment (sarahraqueld) [reply app/dashboards/work_process_dashboard.rb:55]: oops, ok, there's a presence validation on the model, so we're fine

**Comment by sarahraqueld** [comment]
It'd be nice to have a screenshot of how it's looking after the change, so we as reviewers can quickly validate the fix works :)

**Comment by FerPerales** [comment]
@sarahraqueld adding one 😄

**Pull request merged**